### PR TITLE
Move the Application Broker documentation to the Application Connector topic

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

Changes proposed in this pull request:

- Move the AB documents under the Application Connector documentation
- Update the relevant Architecture diagrams
- Update the relevant documents

### CODE REVIEW

File: docs/application-connector/docs/02-01-application-connector.md [hunk 1]
```diff
@@ -37,12 +37,19 @@ An App represents an external solution connected to Kyma. It handles the integra
 Using the components of the Application Connector, the App creates a coherent identity for a connected external solution and ensures its separation.
 All Apps are created through the Application custom resource, which also stores all of the relevant metadata. You can map an App to many Kyma Namespaces and use the APIs and the Event Catalogs of the connected external solution within their context.
 
+# Application Broker
+
+The Application Broker (AB) observes all the Application custom resources and exposes their APIs and Events as ServiceClasses to the Service Catalog. When the list of remote ServiceClasses is available in the Service Catalog, you can create an ApplicationMapping, provision those ServiceClasses, and enable them for Kyma services. This allows you to extend the functionality of existing systems.
+
+The AB implements the [Open Service Broker API](https://www.openservicebrokerapi.org/). For more details about the Service Brokers, see [this](/docs/components/service-brokers#overview-service-brokers) documentation.
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/02-01-application-connector.md]:
```suggestion
The AB implements the [Open Service Broker API](https://www.openservicebrokerapi.org/). For more details about Service Brokers, see [this](/docs/components/service-brokers#overview-service-brokers) documentation.
```

**Comment by tomekpapiernik** [comment]
PS. I think you should assign a milestone to your PR.

File: docs/application-connector/docs/assets/001-application-connector.svg [hunk 2]
```diff
@@ -1,2 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tomekpapiernik) [inline docs/application-connector/docs/assets/001-application-connector.svg]: You edited an outdated version of the diagram - we don't use the term "Remote Environment" anymore.

Comment (klaudiagrz) [reply docs/application-connector/docs/assets/001-application-connector.svg]: Thanks for spotting that 😨

File: docs/application-connector/docs/02-01-application-connector.md [hunk 3]
```diff
@@ -37,12 +37,19 @@ An App represents an external solution connected to Kyma. It handles the integra
 Using the components of the Application Connector, the App creates a coherent identity for a connected external solution and ensures its separation.
 All Apps are created through the Application custom resource, which also stores all of the relevant metadata. You can map an App to many Kyma Namespaces and use the APIs and the Event Catalogs of the connected external solution within their context.
 
+# Application Broker
+
+The Application Broker (AB) observes all the Application custom resources and exposes their APIs and Events as ServiceClasses to the Service Catalog. When the list of remote ServiceClasses is available in the Service Catalog, you can create an ApplicationMapping, provision those ServiceClasses, and enable them for Kyma services. This allows you to extend the functionality of existing systems.
```
[the remark this answers is not in the document]
Comment (klaudiagrz) [reply docs/application-connector/docs/02-01-application-connector.md]: Already discussed with @polskikiel . This sentence is technically correct.

**Pull request merged**